drivers/display/Kconfig.sdl

```diff
@@ -27,4 +27,32 @@ config SDL_DISPLAY_Y_RES
 	int "Y resolution for SDL display"
 	default 240
 
+choice
+	prompt "Default pixel format"
+	default SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_ARGB_8888
+	help
+	  Default pixel format to be used by the display
+
+	config SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_ARGB_8888
+		bool "ARGB 8888"
+
+	config SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_RGB_888
+		bool "RGB 888"
+
+	config SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_MONO01
+		bool "Mono Black=0"
+
+	config SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_MONO10
+		bool "Mono Black=1"
+
+endchoice
+
+config SDL_DISPLAY_DEFAULT_PIXEL_FORMAT
+	int
+	default 1 if SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_RGB_888
```

Comment:
I'm not thrilled about the magic numbers here, but I don't see an alternative.

Could you add a comment here, and at the definition of `enum display_pixel_format`, linking the two by saying that the values of that enumeration must be maintained to be consistent with the values of the `SDL_DISPLAY_DEFAULT_PIXEL_FORMAT` Kconfig entry?

It won't guarantee that the two won't get out of sync, but it could make it less likely.

Reply:
Added some comments but I also don't like the approach.

Alternative would be to use the Kconfig boolean symbols (`SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_...`) in a big `#ifdef` in `sdl_display_init` to assign the corresponding enum value, this would at least guarantee the correct value and we would get rid of the magic numbers at the expense of cluttering the init code.

Reply:
So that might look like:
```
        disp_data->current_pixel_format =
#if defined(SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_RGB_888)
            PIXEL_FORMAT_RGB_888
#elif defined(SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_RGB_MONO01)
            PIXEL_FORMAT_RGB_MONO01
#elif defined(SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_RGB_MONO01)
            PIXEL_FORMAT_RGB_MONO01
#else /* SDL_DISPLAY_DEFAULT_PIXEL_FORMAT */
            PIXEL_FORMAT_RGB_ARGB_8888
#endif /* SDL_DISPLAY_DEFAULT_PIXEL_FORMAT */
            ;
```
I think I like that better.

Reply:
agree, I will take care of it